Documentation/Examples/RetriableSample.dib

```diff
@@ -0,0 +1,660 @@
+#!markdown
+
+# Corvus.Retry.Retriable - Sample Notebook
+
+#!markdown
+
+First, add a reference to the `Corvus.Retry` NuGet package.
+
+#!csharp
+
+#r "nuget: Corvus.Retry, 1.0.2"
+
+#!markdown
+
+Add a using statement for `Corvus.Retry` namespace and the others we'll be using in this notebook.
+
+#!csharp
+
+using Corvus.Retry;
+using Corvus.Retry.Strategies;
+using Corvus.Retry.Policies;
+using System.Net.NetworkInformation;
+using System.Threading;
+using System.Net.Http;
+using System.Web;
+using System.Diagnostics;
+using System.Threading;
+
+#!markdown
+
+`Retriable` is the most common usage pattern from the `Corvus.Retry` library. It wraps an existing method and allows you to re-execute the method if it throws an exception.
+
+```csharp
+Retriable.Retry(() => DoSomethingThatMightFail());
+```
+
+#!markdown
+
+### `Retriable.Retry` – basic example
+
+The following example uses `Retry` to wrap a method that pings an internet host.
+
+#!csharp
+
+Ping pingSender = new Ping();
+Retriable.Retry(() => pingSender.Send("www.endjin.com"));
```

Comment:
does www.endjin.com actually resolve? I thought I got rid of that DNS entry!

Reply:
It has a proper weird response that deeply upsets my browser into thinking it is some kind of attack 😀

Reply:
Probably not a good idea to remove it entirely. Most browsers have a Ctrl-Enter shortcut where if you type, say, `endjin` into the address bar then hit Ctrl-Enter, it navigates to `www.endjin.com`.

I use this a lot because I'm too lazy to type `.com` and then Enter.

Brian mentioned to me last week that he ended up hitting that error page when he tried to go to our site, so apparently I'm not the only one.